homeassistant/components/notify/discord.py

```diff
@@ -22,6 +22,8 @@
     vol.Required(CONF_TOKEN): cv.string
 })
 
+ATTR_IMAGES = 'images'
```

Comment:
What string constant do we use in other notify platforms for image attachment?

Reply:
ATTR_ATTACHMENTS = 'attachments'    in Slack
ATTR_IMAGES = 'images' in Mailgun and SMTP

Reply:
Ok, thanks for the review. Then let's keep it like this.